### ISSUE TITLE
build error in osx 

### ISSUE THREAD

**Issue by riusksk** [google/honggfuzz#62]
The latest version of honggfuzz have the fllowing build error in osx, because "timer_t" is not defined in osx.

```
./common.h:317:9: error: unknown type name 'timer_t'; did you mean 'time_t'?
        timer_t timerId;
        ^~~~~~~
        time_t
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include/sys/_types/_time_t.h:30:26: note: 
      'time_t' declared here
typedef __darwin_time_t         time_t; 
                                ^
1 error generated.
make: *** [honggfuzz.o] Error 1

```

You can add the following code in common.h to fix it:

`typedef long timer_t;`

**Comment by robertswiecki** [comment]
Thanks, fixed it in https://github.com/google/honggfuzz/commit/9f9968af28237233196c2f4574fd348d42730622 - please feel free to test it.

**Comment by riusksk** [comment]
test ok.

**Comment by riusksk** [comment]
This issue can be reproduced in Mac OSX again.
```
./common.h:303:5: error: unknown type name 'timer_t'; did you mean 'time_t'?
    timer_t timerId;
    ^~~~~~~
    time_t
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include/sys/_types/_time_t.h:30:26: note:
      'time_t' declared here
typedef __darwin_time_t         time_t;
                                ^
1 error generated.
make: *** [honggfuzz.o] Error 1
```

**Comment by robertswiecki** [comment]
Thanks for the report - should be fixed in 2c1a691c4c8bb5547b295b1ef00685ec226ad550
